infogami/infobase/_dbstore/store.py

```diff
@@ -72,7 +72,7 @@ def _get_doc(self, row):
         return doc
 
     def put(self, key, doc):
-        if doc.get("_delete") == True:
+        if doc.get("_delete"):
             return self.delete(key, doc.get("_rev"))
```

Comment:
That looks okay, but both have different behavior. The initial code accepts only `True` as a value for `_delete` where as the new one accepts any non-null string or non-zero integer. Do we really want to make these changes in the name of Flake8, without worrying about consequences?

Reply:
Please revert this change.  This PR should only be modifying lines that flake8 is complaining about.